android/filament-android/src/main/java/com/google/android/filament/Renderer.java

```diff
@@ -108,8 +135,8 @@ public Engine getEngine() {
      * @see #endFrame
      * @see #render
      */
-    public boolean beginFrame(@NonNull SwapChain swapChain) {
-        return nBeginFrame(getNativeObject(), swapChain.getNativeObject());
+    public boolean beginFrame(@NonNull SwapChain swapChain, long frameTimeNanos) {
```

Comment:
This is an API breakage. It will impact sceneform. Should we have a fall back API that passes 0?

Reply:
So I broke it on purpose so that they actually plumb things through. But I can be convinced otherwise. I'm afraid if we don't do that, everybody will pass 0.